customitem/README.md

```diff
@@ -0,0 +1 @@
+# Custom Item
```

Comment:
Something descriptive should be written if you're using a `README` file, otherwise just remove the file to use `PLUGIN.description`